examples/imagenet/train_imagenet.py

```diff
@@ -61,7 +61,7 @@ def get_example(self, i):
 
         image = image[:, top:bottom, left:right]
         image -= self.mean[:, top:bottom, left:right]
-        image /= 255
+        image *= (1.0 / 255.0)
```

Comment:
Unrelated to  speed-up, could you add a comment above `# Scale to [0, 1]`. Although trivial, the other pre-processing steps are documented.